enginetest/enginetests.go

```diff
@@ -588,6 +597,8 @@ func TestOrderByGroupBy(t *testing.T, harness Harness) {
 		e := mustNewEngine(t, harness)
 		defer e.Close()
 		ctx := NewContext(harness)
+
+		_, isServerTest := e.(*ServerQueryEngine)
```

Comment:
(minor) might be worth having a helper function for this pattern since it's used commonly. That way, you could just have it return a boolean and might be a little easier/cleaner to use in the code. Seems like it's something we'll need to use in many places.